Sources/Test/SimpleTesting/Partitioned/PartitionedStreamCheckpointTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using Microsoft.StreamProcessing;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SimpleTesting
+{
+    /* This test case verifies fix for
+     * Bug 944724: Exception in Trill SessionWindow when Partition (substreams) is used and restored from checkpoint
```

Comment:
>944724 [](start = 11, length = 6)

Since this is GitHub repo please do not reference internal bugs, just describe them if needed

Reply:
reworded